libxfdashboard/click-action.c

```diff
@@ -887,3 +887,24 @@ void xfdashboard_click_action_release(XfdashboardClickAction *self)
 	_xfdashboard_click_action_set_held(self, FALSE);
 	_xfdashboard_click_action_set_pressed(self, FALSE);
 }
+
+/**
+ * xfdashboard_click_action_is_left_button_or_touch
+ * @self: A #XfdashboardClickAction
+ *
+ * Checks if the specified click action is either a left button press or a single touch 'tap'
+ */
```

Comment:
Please document the return value also even if everybody may know ;)

```
 *
 * Return value: Returns %TRUE if click action event is a left button press or a single touch 'tap', otherwise %FALSE.
```